### PR TITLE
New screen size controls on Display Layout & Effects screen

### PR DESCRIPTION
**Pull request by hrydgard**
I feel the new ones make more sense and are more flexible, and no more confusing "modes".

(post effects UI is still ugly, will clean that up later)

image

(Will fixup the translation strings before I merge).

### CODE REVIEW

File: UI/GamepadEmu.cpp [hunk 1]
```diff
@@ -583,8 +595,10 @@ void PSPCustomStick::Touch(const TouchInput &input) {
 			usedPointerMask &= ~(1 << input.id);
 			analogPointerMask &= ~(1 << input.id);
 			ProcessTouch(input.x, input.y, false);
+			retval = true;
 		}
 	}
+	return true;
```
Comment (unknownbrackets) [inline UI/GamepadEmu.cpp:601]:
```suggestion
	return retval;
```

(or else get rid of retval.)

-[Unknown]

**Review by unknownbrackets — APPROVED** [review]

**Comment by Gamemulatorer** [comment]
Might be related to the user interface changes in graphics settings.
When I play a game and try to change rendering resolution during in game like for example from X2 to X1 rendering resolution doesn't change still X2 applied while in settings is X1 🤔

**Comment by hrydgard** [comment]
Hm, interesting. I'll take a look

**Comment by hrydgard** [comment]
@Gamemulatorer Submitted a fix, seems to work for me.

**Comment by Gamemulatorer** [comment]
> @Gamemulatorer Submitted a fix, seems to work for me.

Thanks

**Pull request merged**